### PR TITLE
Fix configuration with options BUILD_HPP_PLUGIN and BUILD_ROS_INTERFACE

### PR DESCRIPTION
**Pull request by florent-lamiraux**

### CODE REVIEW

**Comment by florent-lamiraux** [comment]
This pull request contains commits of jmirabel/master.

File: CMakeLists.txt [hunk 1]
```diff
@@ -55,13 +55,16 @@ IF(BUILD_ROS_INTERFACE)
   ADD_REQUIRED_DEPENDENCY("agimus_sot_msgs")
 ELSE(BUILD_ROS_INTERFACE)
   ADD_REQUIRED_DEPENDENCY("geometry_msgs")
-  ADD_REQUIRED_DEPENDENCY("roscpp")
 ENDIF(BUILD_ROS_INTERFACE)
 
-ADD_REQUIRED_DEPENDENCY("omniORB4")
-ADD_REQUIRED_DEPENDENCY("hpp-corbaserver")
```
Comment (jmirabel) [inline CMakeLists.txt]: Did you have problem with this two ? The ROS interface needs both of them.

Comment (florent-lamiraux) [reply CMakeLists.txt]: I think I removed them because I though it was useless.
We can reintroduce them.

Comment (jmirabel) [reply CMakeLists.txt]: Fine. I'll take care of this.

**Pull request merged**